src/Dockerfile

```diff
@@ -7,6 +7,7 @@ FROM openjdk:8u171-jdk-stretch as builder
 LABEL maintainer="Victor Romero <victor.romero@gmail.com>"
 
 ARG ACTIVEMQ_ARTEMIS_VERSION
+ARG ACTIVEMQ_DISTRO_URL
```

Comment:
I'm not super convinced of using a contracted form of distribution, as it would be the onl contracted word, i.e: `ARTEMIS_USERNAME` vs ~~`ARTEMIS_USER`~~, `ARTEMIS_PASSWORD` vs ~~`ARTEMIS_PASS`~~